### DISCUSSION TITLE
Image is being distorced

### DISCUSSION THREAD

**Discussion by mgomessp** [arkypita/LaserGRBL#2118] [Q&A]
I made some tests and engraved a square 2 cm X 2 cm and when I´m using a speed even like 2000, 4000 the square is engraved with the measurements distorted when I put a speed odd the square is engraved with the measurements Ok
![20230804_180947](https://github.com/arkypita/LaserGRBL/assets/31015259/c3dce3de-754f-48a4-81f1-e71a05fd2d5e)

**Comment by arkypita** [comment]
Of course it is not a lasergrbl issue.
Speed is just a number that lasergrbl send as-is to the machine controller.

**Reply by arkypita** [reply]
check your machine configuration (especially max speed and acceleration of both axis) and check your assembly

**Reply by mgomessp** [reply]
Isn´t the program that calculate for position and force to send to the control pane ? While the program is running, is been showing the code that is been sending to the control pane, so I thought that the problem can be in the program or some parameter that I need to inform

How do I check this two points that you wrote ?

**Reply by arkypita** [reply]
Why do you need to check?

Download CAMotics https://camotics.org that gives you a preview of the job executed by a laser or cnc, export lasergrbl generated gcode (file - save) and load in CAMotics. You will see that the preview shown by camotics is correct.

**Reply by mgomessp** [reply]
Thank You !

**Comment by arkypita** [comment]
This is a guide to machine parameters.
https://lasergrbl.com/wp-content/uploads/2020/05/Grbl-Configuration-ENG.pdf

Accellerations should stay in a range of 1000-2000 for a safe start/stop and change of direction.
Max speed below 6000.

Those are safety value, if the machine is well trimmed you can use higher values.
Save/export your original configuration before any change.

Hardware assembly: no belt slippage, belt tension, wheel... everything.
also check machine is firm on the table and does not shake on the table when job is in progress
